api/src/main/java/org/openmrs/Order.java

```diff
@@ -363,6 +363,38 @@ public void setCommentToFulfiller(String commentToFulfiller) {
 	}
 	
 	/**
+	 * Convenience method to determine if the order is activated as of the current date
+	 * 
+	 * @return boolean indicating whether the order was activated before or on the current date
+	 * @since 2.0
+	 * @see #isActivated(java.util.Date)
```

Comment:
You should have should to test that this method defaults to now as the check date